.github/config.yml

```diff
@@ -0,0 +1,14 @@
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening your first issue here at xarray! Be sure to follow the issue template!
+  ![coding gif](https://media.giphy.com/media/wv1RNuvWMjQ10bzExO/giphy.gif)
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Thank you for opening this pull request! Please check out our contributing guidelines at https://github.com/pydata/xarray/blob/main/CONTRIBUTING.md .
+  ![cat coding](https://media.giphy.com/media/3oKIPnAiaMCws8nOsE/giphy.gif)
+
+# Comment to be posted to on pull requests merged by a first time user
+firstPRMergeComment: >
+  Congratulations on merging your first pull request! We here at xarray are proud of you!
```

Comment:
```suggestion
  Congratulations on merging your first pull request! Welcome to Xarray!
  We are proud of you, and hope to see you again!
```

Reply:
To be clear, I like the GIF on merge.

The others will be a bit distracting since maintainers open issues and PRs many time.s

Reply:
Okay sure!
I'll keep the gif on the merge and remove the rest then?